client/app/auth/auth.js

```diff
@@ -1,6 +1,6 @@
 angular.module('throwcast.auth', [])
 
 .controller('AuthController', function($scope, $http){
-  $scope.signup = 'Sign-Up';
-  $scope.signin = 'Sign-In';
+  $scope.signup = 'Signup';
```

Comment:
These two variables (`$scope.signup`, `$scope.signin`) don't really make sense. I think they should just be strings in the template.